modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/test/java/com/liferay/portal/search/tuning/rankings/web/internal/BaseRankingsWebTestCase.java

```diff
@@ -90,6 +94,33 @@ public abstract class BaseRankingsWebTestCase {
 	public ComplexQueryPartBuilderFactory complexQueryPartBuilderFactory =
 		Mockito.mock(ComplexQueryPartBuilderFactory.class);
 
+	protected static void setUpServicesForRankingResultUtil() {
```

Comment:
@slnn You can just write `@BeforeClass` and `@AfterClass` annotations in the base class, so that you don't need to write

```
	@BeforeClass
	public static void setUpClass() {
		setUpServicesForRankingResultUtil();
	}

	@AfterClass
	public static void tearDownClass() {
		tearDown();
	}
```
in every sub class.